### PR TITLE
Implemented Heroes Quest

### PR DESCRIPTION
**Pull request by JesseGuerrero**
-Heroes Quest
-Commented out Money Greegree code fodder, kept it because I put some time into it
-Fixed Lost City Bug
-Added Dialogues to a nice number of NPCs
-Added Alfonso shop from Brimhaven using inventory.json from unknown shops folder
-[Ice Queen lair](https://runescape.wiki/images/Ice_Queen%27s_Lair_map.png) added with all mobs
-Changed debug command privileges to be more strict in case servers are accidently in debug mode

### CODE REVIEW

File: src/main/java/com/rs/game/player/content/commands/debug/Debug.java [hunk 1]
```diff
@@ -116,262 +125,262 @@ public static void startup() {
...
-		//
-		// case "delload":
-		// player.deleteLoadout(cmd[1]);
-		// return true;
-		//
-		// case "loadouts":
-		// player.sendLoadoutText();
-		// return true;
-	}
+        Commands.add(Rights.DEVELOPER, "random", "Forces a random event.", (p, args) -> {
```
Comment (titandino) [inline src/main/java/com/rs/game/player/content/commands/debug/Debug.java:128]: These should be player level commands. Debug mode shouldn't be on unless the server is a debug server.

Comment (titandino) [reply src/main/java/com/rs/game/player/content/commands/debug/Debug.java:128]: Also for some reason your IDE is still replacing my tabs with spaces haha

File: src/main/java/com/rs/game/player/content/world/regions/PortSarim.java [hunk 2]
```diff
@@ -21,140 +21,192 @@
...
-								player.getInventory().deleteItem(24344, 1);
-								player.getInventory().deleteItem(24346, 1);
-								player.getInventory().addItem(e.getItem().getId() == 24339 ? 24338 : 24337, 1);
-								player.sendMessage("Thurgo "+(e.getItem().getId() == 24339 ? "repairs" : "forges")+" your Royal crossbow.");
-							}
-					}
-				});
-		}
-	};
+    public static ItemOnNPCHandler handleThurgoItem = new ItemOnNPCHandler(604) {
```
Comment (titandino) [inline src/main/java/com/rs/game/player/content/world/regions/PortSarim.java:44]: My tabs again :(

**Review by titandino — CHANGES_REQUESTED** [review]

**Comment by JesseGuerrero** [comment]
okay making changes now

**Review by titandino — APPROVED** [review]

**Comment by JesseGuerrero** [comment]
was almost done. But yea 
image
I will be more cognizant. This was set to spaces for the auto-formater. Though when I format it tends to change the number of indentations

**Pull request merged**